src/encoded/static/components/cart/status.js

```diff
@@ -117,10 +126,10 @@ class CartStatusComponent extends React.Component {
             return (
                 <NavItem
                     dropdownId="cart-control"
-                    dropdownTitle={<CartNavTitle elements={elements} inProgress={inProgress} />}
+                    dropdownTitle={<CartNavTitle elements={elements} locked={locked} inProgress={inProgress} />}
                     openDropdown={openDropdown}
                     dropdownClick={dropdownClick}
-                    label={`Cart containing ${elements.length} ${elements.length > 1 ? 'items' : 'item'}`}
+                    label={`${locked ? 'locked' : ''} cart containing ${elements.length} ${elements.length > 1 ? 'items' : 'item'}`}
```

Comment:
I was OK with the extra space because you never see this text — you only hear it with a screen reader. That’s why I use the lower-case letters to start the line as well.